src/util/apiHelpers.js

```diff
@@ -74,45 +68,26 @@ const uri = (() => {
   return apiResourceUrl('/graphql-api/graphql');
 })();
 
-// See: https://www.apollographql.com/docs/react/advanced/fragments.html#fragment-matcher
-const fragmentMatcher = new IntrospectionFragmentMatcher({
-  introspectionQueryResultData: {
-    __schema: {
-      types: [
-        {
-          kind: 'INTERFACE',
-          name: 'TaxonomyEntity',
-          possibleTypes: [{ name: 'Resource' }, { name: 'Topic' }],
-        },
-        {
-          kind: 'INTERFACE',
-          name: 'SearchResult',
-          possibleTypes: [
-            { name: 'ArticleSearchResult' },
-            { name: 'LearningpathSearchResult' },
-          ],
-        },
-      ],
-    },
+const possibleTypes = {
+  TaxonomyEntity: ['Resource', 'Topic'],
+  SearchResult: ['ArticleSearchResult', 'LearningpathSearchResult'],
+};
+
+const typePolicies = {
+  SearchContext: {
+    keyFields: object =>
+      object.filters?.length ? object.filters.id : object.path,
+  },
+  GroupSearchResult: {
+    keyFields: object =>
+      object.filters?.length ? object.filters.id : object.path,
+  },
+  Filter: {
+    keyFields: object => `${object.id}+${object.relevanceId}`,
```

Comment:
Skal vi fortsatt bruke samme id for filter? relevanceId er ofte undefined

Reply:
RelevanceId skal egentlig være der, men dersom den er tom meiner eg det tilsvarer kjernestoff: url:relevance:core. Filter skal fases ut i løpet av våren så skader uansett ikkje å ha den der.